### ISSUE TITLE
Staging Review finding: Screen reader users never told when more facilities are loaded

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99049]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** 10-10 Health Apps
**Product:** 10-10CG
**Feature:** Improve Facility Selection

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/locator](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/comprehensive-assistance-for-family-caregivers/apply-form-10-10cg/veteran-information/va-medical-center/locator) after completing a search, you get a `va-radio` group containing radio button tiles for your search results followed by a button labeled "Load more facilities." When they press that button, their screen reader announces: "Loading more facilities, loading progress bar." But no subsequent announcement is made. Without the visual cues that something has changed on the page, you don't know when more facilities are present and when you should start proceeding through the rest of the radio group.

## Recommended action

There are a couple of ways you could address this, either are fine:
- Move keyboard focus to the first new radio button added to the page. That will signal to screen reader users that something new is there. (I don't know this for a fact but you may run into shadow DOM issues trying to send focus to a specific radio button though.)
- Add an `aria-live` region to announce when new facilities have been added, eg. "Five new facilities loaded" or similar.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by allanto-ah** [comment]
Environment: Staging

Verified that screen readers now tell the users when more facilities are loaded. 

One thing that came up was when using VoiceOver on macOS, it would only read "5 more facilities loaded" once even though the user may be clicking the Load more facilities button more than once. In NVDA on Windows, this screen reader would read it each time more facilities are loaded. This might just be a difference in the two screen readers, but has been brought up to the devs to investigate

**Comment by allanto-ah** [comment]
Environment: Staging

Verified that the issue above has been fixed for VoiceOver on macOS
